src/mattext/main.py

```diff
@@ -22,6 +22,9 @@ def __init__(self):
     def run_task(self, run: list, task_cfg: DictConfig, local_rank=None) -> None:
         if "benchmark" in run:
             self.run_benchmarking(task_cfg)
+ 
+        if "classification" in run:
+            self.run_classification(task_cfg)
```

Comment:
also this is technical debt we should remove - in the end, it is just a special case of benchmarking